genai/client.go

```diff
@@ -29,17 +29,22 @@ import (
 
 	gl "cloud.google.com/go/ai/generativelanguage/apiv1beta"
 	pb "cloud.google.com/go/ai/generativelanguage/apiv1beta/generativelanguagepb"
+	gld "github.com/google/generative-ai-go/genai/internal/generativelanguage/v1beta" // discovery client
 
-	"github.com/google/generative-ai-go/internal"
 	"github.com/google/generative-ai-go/internal/support"
 	"google.golang.org/api/iterator"
 	"google.golang.org/api/option"
 )
 
+// version is the current tagged release of this package.
+const version = "v0.11.0"
```

Comment:
There are still two version constants. The one in `genai/internal/version` is different from this one

Reply:
Oh, good point. There's only one now.

Reply:
Oh dang, I already merged it. See forthcoming PR.